gorefs.html

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+---
+
+<div class="container">
+    <h1>GO REFs</h1>
+    <p>This is a collection of references used by the GO Consortium</p>
+
+    {% if jekyll.environment == "development" %}
+        {% unless site.data.gorefs %}
+            <div class="alert alert-warning">
+                <strong>Warning!</strong> No GO_REFs found. Did you run <code>make _data/gorefs.yml</code>?
```

Comment:
@pkalita-lbl Technically, this should never be an issue as it's handled automatically, right?

Reply:
Automatic in the production build and deploy workflow, yes (as outlined in https://github.com/geneontology/geneontology.github.io/issues/507). It's not as easy to automate it for a _local_ development environment. I looked into ways to hook into Jekyll's build process to make it fully automatic, full stop, but it felt a bit like overkill. In the end I figured that requiring a developer to run one `make` command if they want to work on that page isn't a bad compromise.

Reply:
Makes perfect sense--I just wanted to make sure we weren't going to panic Suzi.